common/src/main/scala/models/Platform.scala

```diff
@@ -0,0 +1,26 @@
+package models
+
+import play.api.libs.json._
+
+sealed trait Platform
+
+case object Android extends Platform { override def toString = "android" }
+case object iOS extends Platform { override def toString = "ios" }
+case object WindowsMobile extends Platform { override def toString = "windows-mobile" }
+
+object Platform {
+  def fromString(s: String): Option[Platform] = s match {
+    case "android" => Some(Android)
+    case "ios" => Some(iOS)
+    case "windows-mobile" => Some(WindowsMobile)
```

Comment:
case _ => None missing ? else branch on line 20 is probably dead code since MatchError will be thrown instead None